libs/blocks/tabs/tabs.css

```
@@ -492,6 +498,77 @@
   text-shadow: revert;
 }
 
+/* Segmented Control Variant */
+.tabs.segmented-control .tabList {
+  background: none;
+  box-shadow: none;
+  display: flex;
+  align-items: center;
+  position: relative;
+  border-radius: var(--m-rounded-corners);
+  overflow: hidden;
+}
+
+.tabs.segmented-control .tabList .tab-list-container {
+  background: #f8f8f8;
```

Let's re-use variables to aid future redesign efforts
```suggestion
  background: var(--color-gray-100);
```